actix-web/hello-world/Cargo.toml

```
@@ -0,0 +1,10 @@
+[package]
+name = "hello-world"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+actix-service = "2.0.2"
+actix-web = "4.2.1"
+shuttle-service = { version = '0.7.2', git = "ssh://git@github.com/biryukovmaxim/shuttle.git", branch = "feature/support-actix-web", features = ["web-actix-web"] }
```

Looks good to me too! We should replace this so it's ready for release. If you want to test the example before release, you can use a `.cargo/config.toml` file in the root of the shuttle repo so it uses the current version of `shuttle-service`. There is an example of this [here](https://github.com/shuttle-hq/shuttle/blob/main/CONTRIBUTING.md#running-locally)
```suggestion
shuttle-service = { version = '0.7.2', features = ["web-actix-web"] }
```